src/Integration.UnitTests/Connection/ConnectControllerTests.cs

```diff
@@ -55,11 +54,10 @@ public void TestInit()
             serviceProvider.RegisterService(typeof(ILogger), logger);
 
             this.sonarQubeServiceMock = new Mock<ISonarQubeService>();
-            this.connectionWorkflowMock = new Mock<IConnectionWorkflowExecutor>();
```

Comment:
Miscellaneous: most of the tests don't need the workflow executor, so I removed it from `TestInitialize` and added it to the tests that needed it.